iac-extensions/arm/src/test/resources/checks/IpRestrictedAdminAccessCheck/Microsoft.Network_networkSecurityGroups_securityRules/missing_values.bicep

```diff
@@ -0,0 +1,49 @@
+resource do_not_raise_an_issue_missing_direction 'Microsoft.Network/networkSecurityGroups/securityRules@2022-11-01' = {
+  name: 'Do not raise an issue: missing direction'
+  properties: {
+    access: 'Allow'
+    protocol: 'TCP'
+    destinationPortRange: '22'
+    sourceAddressPrefix: 'Internet'
+  }
+}
+
+resource do_not_raise_an_issue_missing_access 'Microsoft.Network/networkSecurityGroups/securityRules@2022-11-01' = {
+  name: 'Do not raise an issue: missing access'
+  properties: {
+    direction: 'Inbound'
+    protocol: 'TCP'
+    destinationPortRange: '22'
+    sourceAddressPrefix: 'Internet'
+  }
+}
+
+resource do_not_raise_an_issue_missing_protocol 'Microsoft.Network/networkSecurityGroups/securityRules@2022-11-01' = {
+  name: 'Do not raise an issue: missing protocol'
+  properties: {
+    direction: 'Inbound'
+    access: 'Allow'
+    destinationPortRange: '22'
+    sourceAddressPrefix: 'Internet'
+  }
+}
+
+resource do_not_raise_an_issue_missing_destinationPortRange 'Microsoft.Network/networkSecurityGroups/securityRules@2022-11-01' = {
+  name: 'Do not raise an issue: missing destinationPortRange'
+  properties: {
+    direction: 'Inbound'
+    access: 'Allow'
+    protocol: 'TCP'
+    sourceAddressPrefix: 'Internet'
+  }
+}
+
+resource do_not_raise_an_issue_missing_sourceAddressPrefix 'Microsoft.Network/networkSecurityGroups/securityRules@2022-11-01' = {
+  name: 'Do not raise an issue: missing sourceAddressPrefix'
+  properties: {
+    direction: 'Inbound'
+    access: 'Allow'
+    protocol: 'TCP'
+    destinationPortRange: '22'
```

Comment:
See relevant JSON, there is trailing coma what makes invalid syntax

Reply:
But shouldn't the parser be failing then? At least the Bicep parser will fail if I add it here.

Reply:
Bicep file is OK, I just found the mistake in the corresponding JSON file (trailing comma).
I was unable to leave the comment in JSON file, because it wasn't changed in PR.
It looks like the parser doesn't fail (which is not an issue for us).